projects/burnr/src/hooks/api/useApiCreate.ts

```diff
@@ -35,13 +35,15 @@ export default function useApiCreate (): ApiPromise {
   useEffect((): void => {
     const choseSmoldot = async () => {
       try {
-        const chainSpec =  JSON.stringify(westend);
+        const chainSpec = JSON.stringify(westend);
         //TODO: Here we must check if extension exists before choosing
-        const provider = new SmoldotProvider(chainSpec);
-        await provider.connect();
-        await provider.send('system_health', []);
+        const detect = new Detector('westend', chainSpec);
+        const api = await detect.connect();
+        // const provider = new SmoldotProvider(chainSpec);
+        // await provider.connect();
+        // await provider.send('system_health', []);
         // TODO:  API should be included inside the substrate connect (Both SmoldotProvider and extension should return an API most probably)
-        const api = await ApiPromise.create({ provider });
+        // const api = await ApiPromise.create({ provider });
         mountedRef.current && setApi(api);
       } catch (err) {
         console.log('A wild error appeared:', err);
```

Comment:
Remove the `try` / `catch` and let unexpected errors be thrown in the dev console.. that way they will have better context information for debugging and be easier to spot and fix

Reply:
This is not correct. Unexpected error will make the app to break completely. We do not want that

Reply:
Fair enough ... I don't know about the error handling strategy in burnr.  
Not for this PR but shouldn't we be showing some sort of error UI instead of ignoring the error?  Presumably burnr is unusable with a broken API client

Reply:
There is already an open issue for that #86